drivers/gpio/gpio_litex.c

```diff
@@ -38,13 +40,22 @@ struct gpio_litex_cfg {
 
 struct gpio_litex_data {
 	struct gpio_driver_data common;
+	const struct device *dev;
+	sys_slist_t cb;
 };
 
 /* Helper macros for GPIO */
 
 #define DEV_GPIO_CFG(dev)						\
 	((const struct gpio_litex_cfg *)(dev)->config)
 
+/* Helper macros for GPIO interrupts */
+
+#define GPIO_EV_PENDING(dev) \
+		((volatile uint32_t *)((uint32_t)DEV_GPIO_CFG(dev)->reg_addr + 0x0008))
+#define GPIO_EV_ENABLE(dev) \
+		((volatile uint32_t *)((uint32_t)DEV_GPIO_CFG(dev)->reg_addr + 0x000c))
```

Comment:
Same issue as above -- our build uses uint8_t and 0x14 for this register.  Note the CSR is configurable in LiteX, so this type of issue will not be unique to our builds.